src/z3c/saconfig/zcml.py

```diff
@@ -1,111 +1,128 @@
+from .interfaces import IEngineFactory
+from .interfaces import IScopedSession
+from .utility import EngineFactory
+from zope.configuration.name import resolve
+
+import six
+import zope.component.zcml
 import zope.interface
 import zope.schema
-import zope.component.zcml
+
+
 try:
     from zope.component.security import PublicPermission
 except ImportError:
     # BBB for Zope 2.10
     from zope.component.zcml import PublicPermission
-from zope.configuration.name import resolve
 
-import utility
-import interfaces
 
 class IEngineDirective(zope.interface.Interface):
     """Registers a database engine factory."""
 
     url = zope.schema.URI(
-        title=u"Database URL",
-        description=u"e.g. 'sqlite:///:memory:'.",
+        title=six.text_type("Database URL"),
```

Comment:
I think this is not necessary: u"Database URL" is equal to six.text_type("Database URL") in both Python versions

Reply:
You are right in Python 3.3+ it's legal again to use u''. I will revert that